cmd/cli/helpers/todo.go

```diff
@@ -0,0 +1,76 @@
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	"github.com/bacalhau-project/bacalhau/cmd/util/flags/cliflags"
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+func JobToYaml(job *models.Job) (string, error) {
+	yamlBytes, err := yaml.Marshal(job)
+	if err != nil {
```

Comment:
what is the purpose of this file and why is it called `todo.go`?

Reply:
It contains helper methods to be used by various CLIs, just not sure what to name the file - will go with `helpers.go` unless an alternative is proposed here.